src/lib/converter/factories/declaration.ts

```diff
@@ -158,6 +159,20 @@ function setupDeclaration(context: Context, reflection: DeclarationReflection, n
     return reflection;
 }
 
+// we should not be merging type and value with the same name,
+// because TypeScript have different namespaces for these two categories
```

Comment:
Slight grammar adjustment
```suggestion
// because TypeScript has different namespaces for these two categories
```